docs/latest/user/udp-routing.md

```diff
@@ -0,0 +1,196 @@
+# UDP Routing
+
+The [UDPRoute](https://gateway-api.sigs.k8s.io/references/spec/#gateway.networking.k8s.io/v1alpha2.UDPRoute/)
+resource allows users to configure UDP routing by matching UDP traffic and forwarding it to
+Kubernetes backends. This guide will use a CoredDNS example to walk you through the steps required to configure
+ UDPRoute on Envoy Gateway.
+
+## Prerequisites
+
+- A Kubernetes cluster with `kubectl` context configured for the cluster.
+
+__Note:__ Envoy Gateway is tested against Kubernetes v1.24.
```

Comment:
This is no longer the case so please drop L12. I will send a separate PR to remove this line from all examples other than the quickstart. In the quickstart, I will link to the compatibility matrix to avoid having to manage Kube version deps in the user docs.